### ISSUE TITLE
How can I use trained model to test's data without label

### ISSUE THREAD

**Issue by wangchenwei2000** [Microsoft/CNTK#110]
hello All
if I have used CNTK to train models and how can I use this model to data's without label , for example, if I have trained a classification model, I want to use this model to classification some data and output this data formatted,  In the action = "write" , how can I configurate it (exclude to define features,labels)?

thank you very much

**Comment by itguy327** [comment]
Same question here. Thanks in advance for your help

**Comment by dongyu888** [comment]
If your test data is organized similar to the training data you can use the reader section that is similar to the training configuration. The write command allows you write out values of any node by specifying OutputNodeNames. Please note that since label does not exist in this case, you should not have a label section in the reader (if you are using UCIFastReader you need to merge dongyu/UCIFastReaderFix branch to your local branch) and the nodes you want to write information out should not depends on the label node. Also note that:

1. if you just specify the file name as the output, the node info will be written to files in text format.  For speech data, for example, we have HTKMLFWriter and KaldiWriter which will write results into specific binary format for HTK and Kaldi toolkit, respectively. 
2. For classification you can output the node before softmax (which is scaled log posterior probability). However, the output is in the formation of probabilities of each class. So you will need to write a script to parse the output to generate the format of classification result you want. 
3. Someone already wrote a new node to output classification results directly in one of the pull requests. You can merge that fork to your own branch if you don't want to write a script to process the data. Frank will write a better output node for this purpose very soon.
4. If your output requires some precomputed values computed from labels (e.g., prior of classes), then the write command has a bug which I am fixing right now. The fix will be pushed to GitHub in the next several days.

**Comment by wolfma61** [comment]
I close this, since no follow-up question was posted the issue seems to be resolved
